CONTRIBUTING.md

```diff
@@ -177,7 +177,7 @@ libraries *may* implement *proposals*, though are encouraged to not enable the
 ## Stage 2: *Draft*
 
 An RFC at the *draft* stage is a fully formed solution. There is working group
-consensus that the problem identified should be solved, and this particular
+consensus that the problem identified should be solved, and that this particular
```

Comment:
Yes, either this `that` needs to be added, or both `that`s need to be removed (which would be my preference):

```suggestion
consensus the problem identified should be solved, and this particular
```